tests/Lykke.Service.ConfirmationCodes.Tests/unit-test-settings.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Delete this file from git and clear the history. Instruction can be found in confluence.

Reply:
this file is required for tests